docs/code-samples/getting-started/daml/Scripts/Transfer.daml

```diff
@@ -67,8 +67,8 @@ runTransfer = do
     exerciseCmd aliceRequestCid CreateAccount.Accept with
       label = "Alice@Bank"
       description = "Account of Alice at Bank"
-      accountFactoryCid = accountFactoryCid
```

Comment:
I think I had written this on purpose as I didn't know how well-documented / known the short syntax was

Reply:
at some places,  `Foo with a = a; ...` was used but not always, so removed the longer syntax to make it consistent.

Reply:
how about I add a comment?
```
          -- If a variable is in scope that match a field name, we can pick it up implicitly,
          -- writing just `custodian` instead of `custodian = custodian`.
```

Reply:
How about my proposal in the suggestion?

Reply:
I will remove the comment

Reply:
Added a comment for the first occurrence in each file